### PR TITLE
Documentation update for the OIST configuration

### PR DESCRIPTION
**Pull request by charles-plessy**
Hello, this is a documentation update. The main change is the name of the nextflow module on our cluster, and on top of that I took the opportunity to brush up some of the text and add extra information.

### CODE REVIEW

**Review by jfy133 — APPROVED** [review]

File: docs/oist.md [hunk 1]
```diff
@@ -18,16 +18,22 @@ Before running the pipeline you will need to load Nextflow and Singularity
 using the environment module system on _Deigo_. You can do this by issuing the
 commands below:
 
-```bash
+````bash
```
Comment (maxulysse) [inline docs/oist.md:21]:
```suggestion
```bash
```

File: docs/oist.md [hunk 2]
```diff
@@ -18,16 +18,22 @@ Before running the pipeline you will need to load Nextflow and Singularity
...
 ## Load the latest Nextflow and Singularity environment modules
 ml purge
-ml bioinfo-ugrp-modules
-ml Other/Nextflow
-```
+ml bioinfo-ugrp-modules Nextflow2
```
Comment (maxulysse) [inline docs/oist.md:24]:
```suggestion
ml bioinfo-ugrp-modules Nextflow2
```
```

File: docs/oist.md [hunk 3]
```diff
@@ -18,16 +18,22 @@ Before running the pipeline you will need to load Nextflow and Singularity
...
+
+If you run a _nf-core_ pipeline via the `nf-core` tool, the following command
+will load all the modules you need.
+
+```bash
+ml bioinfo-ugrp-modules nf-core
+````
```
Comment (maxulysse) [inline docs/oist.md:31]:
```suggestion
```
```

**Review by maxulysse — APPROVED** [review]

**Comment by maxulysse** [comment]
prettier actually messed up the markdown, but I fixed it

**Pull request merged**